src/filesystem/FuseInterface.cpp

```diff
@@ -334,6 +340,33 @@ FuseInterface::operator()(const fs::path& mntpoint,
                             }
                         }));
 
+    network_thread = boost::thread([&]
+                {
```

Comment:
The thread should only be started if network_server_ != nullptr, otherwise the VERIFY will fire and take down the process if built with assertions enabled.